packages/react-tinacms-editor/src/state/plugins/Menu/Menu.tsx

```diff
@@ -86,7 +86,11 @@ const LinkControl = markControl({
   isDisabled: (view: EditorView) => {
     const { state } = view
     const { selection, schema } = state
-    const linkPresent = selection.$anchor
+    const { anchor, head } = view.state.selection
+    let pos = anchor < head ? anchor : head
+    pos = selection.empty ? pos : pos + 1
```

Comment:
This is interesting. Two questions:

Do you know why the `anchor` might be before the `head` sometimes?

In what situation would the selection be empty? Is it when you click at the start of the link?

Reply:
Hey @ncphillips : anchor < head is test is selected in reverse direction like this: 
![sel](https://user-images.githubusercontent.com/2182307/76522805-3d4f6080-648d-11ea-8431-c06478f55f09.gif)

Selection is `empty` in prose-mirror when its only 1 position, its not extended over text. It is always empty in case of click.

Reply:
Oh wicked! Thanks 🙌